cardano-constitution/test/Cardano/Constitution/Validator/Data/GoldenTests/unsorted.uplc.golden

```diff
@@ -1376,18 +1341,15 @@ program
               force ifThenElse (equalsInteger x y) (constr 0 []) (constr 1 [])))
         (\eta eta ->
            force
-             (force
-                (force ifThenElse
-                   (equalsInteger eta eta)
-                   (delay (delay (constr 0 [])))
-                   (delay
-                      (delay
-                         (force
-                            (force
-                               (force ifThenElse
-                                  (lessThanEqualsInteger eta eta)
-                                  (delay (delay (constr 2 [])))
-                                  (delay (delay (constr 1 []))))))))))))
+             (force ifThenElse
+                (equalsInteger eta eta)
+                (delay (constr 0 []))
+                (delay
+                   (force
+                      (force ifThenElse
+                         (lessThanEqualsInteger eta eta)
+                         (delay (constr 2 []))
+                         (delay (constr 1 []))))))))
```

Comment:
Instead of trying to make case-of-case clever, we should just propagate `force` into the branches whenever appropriate, because that covers more cases, like the one we see here for example.